testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/validation/ValidationXMLTest.java

```diff
@@ -438,4 +432,15 @@ private static <T> Collection<String> resolveValues(final Collection<T> c, final
                 .map(mapper)
                 .collect(Collectors.toList());
     }
+
+    private static boolean hasItems(Collection<String> items, String... controlList) {
+        for (String c : items) {
+            for (String s : controlList) {
+                if (c.equals(s)) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
```

Comment:
This doesn't look quite right as it will exit on the first match. This could probably just be:
```java
private static boolean hasItems(Collection<String> items, String... controlList) {
    return items.containsAll(List.of(controlList));
}
```

Reply:
good catch.  Fixed.